src/renderer/base/FontInfoDesired.cpp

```diff
@@ -8,23 +8,24 @@
 FontInfoDesired::FontInfoDesired(const std::wstring_view& faceName,
                                  const unsigned char family,
                                  const unsigned int weight,
-                                 const til::size coordSizeDesired,
+                                 const float fontSize,

```

Comment:
how does this work with GDI in conhost?

Reply:
The constructor converts back and forth between the existing `til::size` and new `float` fontSize. conhost specifically doesn't use this constructor, but rather the `FontInfo` constructor below.